modules/external-auth-configuring.adoc

```diff
@@ -24,6 +20,7 @@ This procedure uses Keycloak as the identity provider and assumes that you have
 
 * You have access to the `kubeconfig` file generated by the installation program for the cluster.
 * You have backed up any existing authentication configuration, in case you need to revert back to using the built-in OAuth server for authentication.
+* You have the {product-title} web console enabled on the cluster.
```

Comment:
@bergerhoffer , your latest push includes this line change. This is good for the page's example. But this will confuse users. They will think you mean external oidc is not applicable for the cluster if console is disabled. External oidc is applicable when console is disabled, where the configuration just shouldn't include the console oidcClients and the configuration with CLI oidcClients for successful oc login is still applicable.
So could you improve accordingly to get rid of users/readers' such confusion.